snomed/com.b2international.snowowl.snomed.api.rest/src/com/b2international/snowowl/snomed/api/rest/SnomedReferenceSetMemberRestService.java

```diff
@@ -198,6 +199,10 @@ public SnomedReferenceSetMemberRestService() {
 		final String userId = principal.getName();
 		
 		final SnomedRefSetMemberRestInput change = body.getChange();
+		
+		final String id = change.getId();
+		UUID.fromString(id);
```

Comment:
Is this method call here to check if the format is correct? Please add a comment if so.